README.md

```diff
@@ -10,7 +10,7 @@
 # IBM Cloud Security & Compliance Center Go SDK Version v5.0.0
 
 Go client library to interact with various
-[IBM Cloud Security & Compliance Center APIs](https://cloud.ibm.com/docs?tab=api-docs&category=platform_services%2Csecurity).
+[IBM Cloud Security & Compliance Center APIs](https://cloud.ibm.com/apidocs/security-compliance).
```

Comment:
Can't use `&` change IBM Cloud Security and Compliance Center APIs

Reply:
fixed